framework/resources/js/query.js

```diff
@@ -228,12 +228,12 @@
 				options.elements.pagination.on( 'click', '.fw-query-pagination-btn', function ( e ) {
 					let new_page;
 					
+					// Scroll to top of query container.
+					$( this ).closest( '.fw-container' )[0].scrollIntoView( {behavior: 'smooth'} );
```

Comment:
Verify that the item exists before calling `scrollIntoView`.

For example, in the "Explore Maps" page, when you open the list of variables, this new code now prevents the buttons "next" and "previous" from working

![image](https://github.com/CanadianClimateDataPortal/climatedata-wp-theme/assets/159493515/14aa9d5b-d624-42e2-969b-eb66ec1bdfbc)

Reply:
This has been fixed.